cogs/modmail.py

```diff
@@ -2186,6 +2187,56 @@ async def isenable(self, ctx):
 
         return await ctx.send(embed=embed)
 
+    @commands.command(name="export")
+    @checks.has_permissions(PermissionLevel.ADMINISTRATOR)
+    async def export_backup(self, ctx, collection_name):
+        """
+        Export a backup of a collection in the form of a json file.
```

Comment:
As noted in my other comment, we should store the backup as BSON file instead of JSON. Please change the references to BSON files instead.